example/tests/test_shop.py

```diff
@@ -101,6 +107,9 @@ def create_products(self):
         )
         ProductPage.objects.create(page=self.shop_page, product=sdxc_pro_32gb)
         ProductPage.objects.create(page=self.smartcards_page, product=sdxc_pro_32gb)
+        
+        reload_urlconf(urlconf=settings.ROOT_URLCONF)
```

Comment:
with cms34:
```
>>>response = client.get(smartcards_page.get_absolute_url())
Application url changed and urls_need_reloading signal fired. Please reload the urls.py or restart the server.
```

with cms35:
it does not have this red message, but the url must be restarted otherwise it does not find the url of the products

does this imply that there would be a problem with the signal ApphookReloadMiddleware?